strictdoc/backend/sdoc/grammar/type_system.py

```diff
@@ -38,12 +38,12 @@
 ;
 
 FileEntry[noskipws]:
-  ('  FORMAT: ' file_format = FileEntryFormat '\n')?
-   '  VALUE: ' file_path = /.*$/ '\n'
+  (/[ \-]/ ' FORMAT: ' file_format = FileEntryFormat '\n')?
```

Comment:
Just to make sure I understand this:

For FileEntry you always have this in your tests:

```
- TYPE: BibRef
  FORMAT: String
  VALUE: SampleCiteKeyStringRef-1, "The sample BibReference String-Format"
```

and for the `BibEntry` below you have this:

```
- FORMAT: String
  VALUE: RFC-0000, Testing only
```

Would it make sense to hardcode the order of the fields by removing the flexibility: so the first field always has `- ` hardcoded and the second field has a double-space `  `?

Reply:
My context here: I would like to avoid creating opportunities for undefined behavior? What if both components are specified without `- `? In that case, the grammar will look strange but will still work - and that's something that I would like to avoid. Fixing the order of the fields should mostly solve such cases.

Reply:
Wanted to reuse the FileEntry and BibEntry classes.
Maybe I can make the grammar regex parsing more explicit to avoid the undefined behavior.
Also have to add more testcases.